cookbooks/aws-parallelcluster-platform/test/controls/disable_services_spec.rb

```diff
@@ -9,7 +9,7 @@
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 
-control 'tag:config_services_disabled_on_debian_family' do
+control 'tag:testami_tag:config_services_disabled_on_debian_family' do
```

Comment:
we need to enable execution of "tag:testami at build image time in the image builder component.